docs/modding/suitmodding/addingblobs.md

```diff
@@ -0,0 +1,253 @@
+::: 
+
+5. You can then start searching for your suit below in the *Unmodified section*.
+6. Double check if you have the right suit selected by looking up the image in **GlacierKit**.
+To do so, simply 
+
+   - find the correct image code line on the right that starts with ``"Image":".....``
+ 
+   - and locate its **assembly path** 
+
+
+
+>The **assembly path** is the line after ``"Image":"..."`` inside the quotation marks.
+
+     Example of a full image code: 
+   ```json
+   "Image": "images/unlockables_override/47_outfits_bloodmoney_gloves.jpg"
+   ```
+   and its **assembly path**
+   ```json
+   images/unlockables_override/47_outfits_bloodmoney_gloves.jpg
+   ```
+
+- Copy it and paste the **assembly path** in the search bar on top in the *game contents* tab on the left 
+- and hit enter on your keyboard.
+
+
+If the image you found matches the suit you want to modify, you can go back to your **repository** file. (Where you saw the **repository** code).
+
+>Leave open your **GlacierKit** program
+
+:::info GlacierKit vs. RPKG
+While **RPKG** is helpful for viewing images in the **repository**, it’s best to work primarily in **GlacierKit**, as it will be your main modding tool. The **repository code lines** in both tools are identical for each file, so practicing in **GlacierKit** is encouraged for efficient modding.
+:::
+
+
+---
+## STEP 2: **FIND THE IMAGE**
+In this step we will have to look for the correct image code and find its **assembly path**.
+:::note
+You already did this step in the **GlacierKit** method but follow along
+:::
+   ### 1. Open your file
+Click on your suit and you should see its **repository** code on the right
+
+   ### 2. Locate the assembly path
+There will be a line of code starting with `"Image":"....`
+
+   For example:  
+   ```json
+   "Image": "images/unlockables_override/47_outfits_bloodmoney_gloves.jpg"
+   ```
+
+It consists of a
+- **file path** `images/unlockables_override/`
+- **file name** `47_outfits_bloodmoney_gloves.jpg`
+   
+   These 2 together are called the **assembly path** of your file's image.
+
+:::warning
+Keep **RPKG** and/or **GlacierKit** open, you’ll need to view the **assembly paths**.
+:::
+
+---
+
+## STEP 3: **SET UP A BLOBSFOLDER**
+In this step, you’ll create a custom folder structure that ***recreates*** the default image's **assembly path**.
+
+### 1. Make a blobsfolder
+
+   Create a new folder called ***`blobs`*** in your mod's root directory, where the manifest file is located. You can do this in **GlacierKit** or **File Explorer**.
+
+### 2. Recreate the file path
+
+    Create folders that match the **file path** of the image you want to replace.  
+    For example, if the **file path** is `images/unlockables_override/`, your `blobs` folder structure should look like this:
+   ```
+   blobs/images/unlockables_override/
+   ```
+### 3. Add your custom image  
+   - Place your image in the last folder you created (in this example, `unlockables_override`). 
```

Comment:
I can't see this fully previewed obviously but I feel like this formatting will look odd, having a big size-3 heading with a dot point; usually you'd just have body text instead of placing the step in a dot point (also the previous steps don't use dot points).

Reply:
Yes, not being able to preview the site is a problem. Lots of choices are made aren't reflected properly from reviewing the MD file. I was looking into showing and sharing a link to a build version